dao-client/.env.sample

```diff
@@ -0,0 +1,9 @@
+# Path to the secret key of the account that will be used to deploy the contracts.
```

Comment:
I think that this file should already include sane defaults for uncommented variables.

```bash
just prepare-client
```
Should would run `cp .env.sample .env` and maybe print something like "Using defaults from .env.sample"